flink/v1.15/flink/src/test/java/org/apache/iceberg/flink/source/TestStreamingMonitorFunction.java

```diff
@@ -208,6 +211,81 @@ public void testCheckpointRestore() throws Exception {
     }
   }
 
+  @Test
+  public void testInvalidMaxSnapshotCountPerMonitorInterval() {
+    final ScanContext scanContext1 = ScanContext.builder()
```

Comment:
Nit: We normally don't use `final` for variables inside of methods. Is this usage of `final` necessary?

Reply:
Not necessary, removed.